### ISSUE TITLE
App Content 

### ISSUE THREAD

**Issue by Jaylukmann** [hngi/Team-EOS-Mini-Classroom#6]
Lets discuss issues on the content of the EOS classroom App here.

**Comment by DwiseCodeDev** [comment]
Created Content for Mini Class About
Link submitted as text file to Branch Master

**Comment by Ore-cathy** [comment]
can anyone suggest a site we can get nice pictures to fit in?

**Comment by DwiseCodeDev** [comment]
Check here @Ore-cathy https://www.pexels.com/search/design/

**Comment by Ore-cathy** [comment]
alright @DwiseCodeDev

**Comment by Jaylukmann** [comment]
@Ore-cathy were you able to find the pics there?
If not check collectui.com ,pixabay.com,unsplash.com and behance.com

**Comment by DwiseCodeDev** [comment]
Design Elements of the Mini Classroom project from Design Team led by @StephanieOgbudu added to the repo by @Jaylukmann

**Comment by Jaylukmann** [comment]
Yes,I volunteered to help her push to the repo because @StephanieOgbudu,the design team lead was having some glitches with her github.
